test/internet/test-dns-ipv6.js

```diff
@@ -40,6 +40,11 @@ function checkWrap(req) {
   assert.ok(typeof req === 'object');
 }
 
+if (!common.hasIPv6) {
```

Comment:
if it is put ahead of some of the variable declarations in the file then the process exit event won't pass.